ksql-engine/src/test/java/io/confluent/ksql/integration/WindowingIntTest.java

```diff
@@ -65,7 +65,7 @@ public void shouldAggregateTumblingWindow() throws Exception {
 
     // not really required - but lets mess with  ms
     Thread.sleep(100);
-    testHarness.publishTestData(topicName, dataProvider);
+    testHarness.publishTestData(topicName, dataProvider, null);
```

Comment:
Can you get rid of the sleeps in this class now?

Reply:
good spot